extensions/interactions/LogicProof/static/js/sharedSpec.ts

```diff
@@ -17,7 +17,7 @@
  * interaction.
  */
 
-errorWrapper = function(
+var errorWrapperForSharedSpec = function(
```

Comment:
Ditto

Reply:
This cannot be moved out of the global scope since it is used in multiple describe blocks.